src/client/components/utils/ToggleGroup.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Toggle from './ToggleButton';
+
+class ToggleGroup extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = { selected: props.selected };
+        this.changeActive = this.changeActive.bind(this);
+    }
+
+    isActive(i) {
+        if ( this.state.selected === i) {
+            return 'active';
+        } else {
+            return '';
+        }
+    }
+
+    changeActive(i) {
+        this.setState({selected:i});
+    }
+
+    renderButtons() {
+        return this.props.buttons.map((button, index ) => {
+            if ((this.props.buttons.length-1) === index) {
+                return <Toggle updateSearchState={this.props.updateParent} key={index} index={index} text={button.title} addClass={this.isActive(index)} updateGroup={this.changeActive} />
+            } else {
+                return <Toggle updateSearchState={this.props.updateParent} key={index} index={index} text={button.title} addClass={this.isActive(index)+' mr1'} updateGroup={this.changeActive} />
```

Comment:
@vitkuz so you do return the same Toggle but with different class, right? It's a lot easier to solve this problem with css. In case you still want to do it in code I would separate className generation and renderButtons function. In that case you won't need if/else